core/tests/test_utils.py

```diff
@@ -161,6 +164,127 @@ def check_image_png_or_webp(image_string):
     return False
 
 
+class EmailMessageMock(python_utils.OBJECT):
+    """Mock for mailgun API message."""
+
+    def __init__(
+            self, sender_email, recipient_email, subject, plaintext_body,
+            html_body, bcc=None, reply_to=None):
+        self.sender = sender_email
+        self.to = recipient_email
+        self.subject = subject
+        self.body = plaintext_body
+        self.html = html_body
+        self.bcc = bcc
+        self.reply_to = reply_to
+
+
+class EmailServicesMock(python_utils.OBJECT):
+    """Mock for mailgun API"""
```

Comment:
Shouldn't this be a stub for the email service (which has different implementations on dev and prod), rather than the mailgun API?